components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/api/client/GradleEnterpriseApiClient.java

```diff
@@ -37,33 +38,59 @@ public class GradleEnterpriseApiClient {
     private final ConsoleLogger logger;
 
     public GradleEnterpriseApiClient(URL baseUrl, CustomValueNames customValueNames, ConsoleLogger logger) {
+        this.logger = logger;
+        this.baseUrl = baseUrl;
         this.customValueNames = customValueNames;
+
```

Comment:
Assign them in the same order as passed in to the constructor.

Reply:
✅ Done